### PR TITLE
https://issues.redhat.com/browse/ACM-6223 Update late binding 2.6

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

**Comment by cadawson** [comment]
Hi @jelkosz and Vlad. Please take a look at this PR to see if the update looks ok. I can't add Vlad as a reviewer in GH, so will mention in the Jira issue. Thanks!

File: multicluster_engine/cluster_lifecycle/create_infra_env.adoc [hunk 1]
```diff
@@ -12,7 +12,7 @@ You can use the console to create an infrastructure environment to manage your h
 Infrastructure environments support the following features:
 
 - Zero-touch provisioning of clusters: Deploy clusters using a script. See https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html/scalability_and_performance/ztp-deploying-disconnected[Deploying distributed units at scale in a disconnected environment] in the {ocp} documentation for more information.
-- Late binding: Enable the host to be booted by an infrastructure administrator. The creator of a cluster can bind a cluster to that host at a later time. The cluster creator does not have to have administrator privileges to the infrastructure when using late binding.
+- Late binding: Delay binding a cluster to a host. An infrastructure administrator can boot the host, and the creator of a cluster can bind the host to the exisitng cluster at a later time. The cluster creator does not need administrator privileges to access the infrastructure when using late binding.
```
Comment (vkolodny) [inline multicluster_engine/cluster_lifecycle/create_infra_env.adoc]: The same issue in the first sentence. Should be :Delay binding a host to a cluster.
In addition, I assume the explanation can be Like:
Late binding: Enable to add additional node to the existing cluster. An infrastructure administrator can boot the host, and the creator of a cluster can bind the host to the existing cluster at a later time ...

Also - please fix the "exsitng" cluster to existing (misspell)

**Comment by cadawson** [comment]
Hi @vkolodny. Made the review changes. Please take a look. Thanks!

File: multicluster_engine/cluster_lifecycle/create_infra_env.adoc [hunk 2]
```diff
@@ -12,7 +12,7 @@ You can use the console to create an infrastructure environment to manage your h
 Infrastructure environments support the following features:
 
 - Zero-touch provisioning of clusters: Deploy clusters using a script. See https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html/scalability_and_performance/ztp-deploying-disconnected[Deploying distributed units at scale in a disconnected environment] in the {ocp} documentation for more information.
-- Late binding: Enable the host to be booted by an infrastructure administrator. The creator of a cluster can bind a cluster to that host at a later time. The cluster creator does not have to have administrator privileges to the infrastructure when using late binding.
+- Late binding: Add a node to an existing cluster. An infrastructure administrator can boot the host, and the creator of a cluster can bind the host to the existing cluster at a later time. The cluster creator does not need administrator privileges to access the infrastructure when using late binding.
```
Comment (dockerymick) [inline multicluster_engine/cluster_lifecycle/create_infra_env.adoc]:
```suggestion
- Late binding: Add a node to an existing cluster. An infrastructure administrator can boot the host, and the creator of a cluster can bind the host to the existing cluster at a later time. Administrator privileges are not required for the cluster creator to access the infrastructure when using late binding.
```

Comment (dockerymick) [reply multicluster_engine/cluster_lifecycle/create_infra_env.adoc]:
```suggestion
- Late binding: Add a node to an existing cluster. As an infrastructure administrator, you can boot the host, and the cluster creator can bind the host to the existing cluster at a later time. Administrator privileges are not required for the cluster creator to access the infrastructure when using late binding.
```

Comment (dockerymick) [inline multicluster_engine/cluster_lifecycle/create_infra_env.adoc]:
```suggestion
- Late binding: Add a node to an existing cluster. An infrastructure administrator can boot the host, and the cluster creator can bind the host to the existing cluster at a later time. The cluster creator does not need administrator privileges to access the infrastructure when using late binding.
```

**Review by dockerymick — COMMENTED** [review]
made some suggestions

File: multicluster_engine/cluster_lifecycle/create_infra_env.adoc [hunk 3]
```diff
@@ -12,7 +12,7 @@ You can use the console to create an infrastructure environment to manage your h
 Infrastructure environments support the following features:
 
 - Zero-touch provisioning of clusters: Deploy clusters using a script. See https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html/scalability_and_performance/ztp-deploying-disconnected[Deploying distributed units at scale in a disconnected environment] in the {ocp} documentation for more information.
-- Late binding: Enable the host to be booted by an infrastructure administrator. The creator of a cluster can bind a cluster to that host at a later time. The cluster creator does not have to have administrator privileges to the infrastructure when using late binding.
+- Late binding: Add a node to an existing cluster. An infrastructure administrator can boot the host, and the creator of a cluster can bind the host to the existing cluster at a later time. Administrator privileges are not required for the cluster creator to access the infrastructure when using late binding.
```
Comment (dockerymick) [inline multicluster_engine/cluster_lifecycle/create_infra_env.adoc]:
```suggestion
- Late binding: Add a node to an existing cluster. An infrastructure administrator can boot the host, and the cluster creator can bind the host to the existing cluster at a later time. Administrator privileges are not required for the cluster creator to access the infrastructure when using late binding.
```

**Review by dockerymick — APPROVED** [review]
made a comment

**Comment by cadawson** [comment]
HI @vkolodny. If these changes look ok, I will port them forward and we can close the issue. Thanks!

**Comment by dockerymick** [comment]
Approval from development and peer review. Merging this and will make changes to 2.7 and 2.8

**Pull request merged**